### ISSUE TITLE
doc: FAQ entry explaining Go is spelled Go, not Golang

### ISSUE THREAD

**Issue by mdempsky** [golang/go#27616]
This seems to be a recurring point of confusion. Maybe a FAQ entry on correct usage would help.

**Comment by myitcv** [comment]
Agreed. The FAQ should also cover why the word "golang" appears in some places, e.g. golang-nuts, golang-dev etc.

**Comment by mvdan** [comment]
Would also be great to clarify that it can never be spelled "GO". It hurts every single time I read that online.

**Comment by bcmills** [comment]
(CC @robpike)

**Comment by josharian** [comment]
It is worth noting that the new logo isn’t helping; visually, it appears to be two upper case letters.

**Comment by OneOfOne** [comment]
@josharian I mentally blocked the new logo / artwork, but yeah you do have a valid point.

**Comment by cznic** [comment]
> Would also be great to clarify that it can never be spelled "GO". It hurts every single time I read that online.

I concur. And even more when Google does it by itself.

![image](https://user-images.githubusercontent.com/729580/45585103-21dab800-b8df-11e8-8cab-574914bd9d62.png)

**Comment by cznic** [comment]
Deleted my post, @josharian already said the same.

**Comment by dmitshur** [comment]
For anyone who needs to link to this in advance of next Go release, it shows up at tip.golang.org already:

https://tip.golang.org/doc/faq#go_or_golang
